codegen-client/src/test/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/EndpointTraitBindingsTest.kt

```diff
@@ -67,7 +67,7 @@ internal class EndpointTraitBindingsTest {
                 }
                 """,
             )
-            implBlock(model.lookup("test#GetStatusInput"), sym) {
+            implBlock(sym.toSymbol(model.lookup("test#GetStatusInput"))) {
```

Comment:
It feels weird to call `toSymbol` on something already called `sym`. Is there a better name?

Reply:
Renamed `sym` to `symbolProvider`.